docs/introduction/about.md

```diff
@@ -0,0 +1,7 @@
+---
+title: StoneSoup Infrastructure Deployments
+---
+
+Welcome to **StoneSoup Infrastructure Deployments** documentation!  
+
+In the next pages you'll learn how to deploy, test and develop StoneSoup!
```

Comment:
Is there any more information/context that we can provide here? A general or brief overview of the project and what we are achieving?

Should we tell users how to navigate the documentation site? How the previous/next modules will be linked to at the bottom of each page? What about the header links? 

I didn't notice initially that the page links were at the bottom of the pages. Can we invite the user to click the next section at the bottom to draw their attention to it? Should we highlight the navigation pane for the additional navigation help?

Is this where we should tell people where they can reach out for help/information, direct them to the issues board, CONTRIBUTING, or FAQs?

Reply:
I would love to provide a link to the user documentation, but currently, its site isn't public.
I think the layout is pretty straightforward, users should know how to navigate in the docs without further explanation.

Currently, a contributing doc is missing. We can add it in another PR.
The FAQ is linked in the sidebar.
I've added to the top bar a link to the platform channel on slack.